app/helpers/helpers.rb

```diff
@@ -0,0 +1,15 @@
+class Helper < ApplicationController
+
```

Comment:
It would be easier to leave some of these methods in your ApplicationController 

```ruby 
helpers do 
   def logged_in?

  end 

  def current_user 

  end 

end
```